### PR TITLE
FMO-67 FMO-61 FMO-57 FMO-68

### PR DESCRIPTION
**Pull request by jsvapiav**
FMO-67: Update DPF rules right after any changes
FMO-61: SSH login without user interaction
FMO-57: IP address workaround
FMO-68: Require root for ssh and all config manipulation

### CODE REVIEW

File: apps/devicepassthrough_apps.py
Comment (buianhhuy96) [inline apps/devicepassthrough_apps.py]: I think this is a typo, is it?

File: utils/fmoconfig.py
Comment (buianhhuy96) [inline utils/fmoconfig.py:77]: I notice the `write_vmpf_config()` is called in DPF functions,
Should `write_vmddp_config()` be re-written the same way, only in DDP functions? 
It is now inside `save_config()`, which write the ddp config every time there're changes in the config, regardless what the changes are.

File: apps/devicepassthrough_apps.py
Comment (jsvapiav) [reply apps/devicepassthrough_apps.py]: Heh, good catch -- rebase leftovers, will fix it now!

File: utils/fmoconfig.py
Comment (jsvapiav) [reply utils/fmoconfig.py:77]: Yes, you should update your config only when it has changed

Comment (buianhhuy96) [reply utils/fmoconfig.py:77]: Thanks, I'll mark this approve and make that change once this is merged

**Review by buianhhuy96 — APPROVED** [review]

**Pull request merged**
